core/src/main/resources/jenkins/model/BuildDiscarderProperty/help.html

```diff
@@ -1,19 +1,51 @@
 <div>
-  This controls the disk consumption of Jenkins by managing how long you'd like to keep
-  records of the builds (such as console output, build artifacts, and so on.)
-  Jenkins offers two criteria:
-
+  This determines when, if ever, build records for this project should be
+  discarded. Build records include the console output, archived artifacts, and
+  any other metadata related to a particular build.
+  <p>
+  Keeping fewer builds means less disk space will be used in the <i>Build Record
+  Root Directory</i>, which is specified on the <i>Configure System</i> screen.
+  <p>
+  <!-- TODO: Most/all of the following should be dynamic, showing info from the
+       `BuildDiscarder` implementations that are currently loaded -->
+  Jenkins offers two options for determining when builds should be discarded:
```

Comment:
It's mentioned right there in the TODO. At some point this should be fixed to fetch the documentation from the implementations.
The fact that it's an "extension point" is irrelevant to users, unless we have an explicit section in the plugin manager for BuildDiscarders that we can point to.
On 17 Mar 2016 08:14, Oleg Nenashev <notifications@github.com> wrote:In core/src/main/resources/jenkins/model/BuildDiscarderProperty/help.html:
> @@ -1,19 +1,51 @@
>  <div>
> -  This controls the disk consumption of Jenkins by managing how long you'd like to keep
> -  records of the builds (such as console output, build artifacts, and so on.)
> -  Jenkins offers two criteria:
> -
> +  This determines when, if ever, build records for this project should be
> +  discarded. Build records include the console output, archived artifacts, and
> +  any other metadata related to a particular build.
> +  <p>
> +  Keeping fewer builds means less disk space will be used in the <i>Build Record
> +  Root Directory</i>, which is specified on the <i>Configure System</i> screen.
> +  <p>
> +  <!-- TODO: Most/all of the following should be dynamic, showing info from the
> +       `BuildDiscarder` implementations that are currently loaded -->
> +  Jenkins offers two options for determining when builds should be discarded:

This is an extension point, needs to be mentioned. Currently it smells like 2 hardcoded strategies

—You are receiving this because you authored the thread.Reply to this email directly or view it on GitHub